oluseyianani/stackoverflow-lite
# #160269817 Enhance database structure and Route testing

#### What does this PR do?
- Change database structure to be more optimal with how data are stored
- Adjust codebase to pass testing on routes
#### Description of Task to be completed?
Assign appopirate constraints to database vairables
#### How should this be manually tested?
After running script to create the database, store items in the database with expected and not expected data types to see how the database responds.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#160269817
